Core/include/Acts/Geometry/ConeVolumeBounds.hpp

```diff
@@ -34,7 +35,7 @@ class PlanarBounds;
 class ConeVolumeBounds : public VolumeBounds {
  public:
   /// @enum BoundValues for readability
-  enum BoundValues : unsigned int {
+  enum BoundValues : std::uint8_t {
```

Comment:
Is this connected to the projector change? I believe we don't store this often so the memory size shouldn't really matter should it?

Reply:
I put these mostly as a reminder. I don't intent to change those in this PR because that might be breaking.

Do you think these should generally change to be lower size? As you say this will not really matter but I feel like if the box can be smaller it should be smaller.

Reply:
In some cases uint8_t handling is inefficient, but I have no idea whether the type of the enum has some actual consequences other than casts.

Reply:
if it would be `uint8_t` we could pull the type through into the `std::array` later but right now it is too big. with operations I expect `uint8_t` to be a bit slower but if it just sits there I would expect similar performance